lib/index.js

```diff
@@ -33,13 +89,18 @@ function start() {
   }
   const port = findPort();
   const p = spawn(process.execPath, [require.resolve('./worker'), port, ...FLAGS], {
-    stdio: 'inherit',
+    stdio: debugMode ? ['ignore', outputFile, errorFile] : 'inherit',
```

Comment:
Why is this different for debugMode?

Reply:
In debugMode we are logging the output in worker thread into the files, otherwise, we don't log.